### PR TITLE
GEODE-5971: Refactor durable client commands to extend GfshCommand

### PR DESCRIPTION
**Pull request by jdeppe-pivotal**
Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [ ] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [ ] Is your initial contribution a single, squashed commit?

- [ ] Does `gradlew build` run cleanly?

- [ ] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, you check travis-ci for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

File: geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/CloseDurableCQsCommand.java [hunk 1]
```diff
@@ -55,29 +55,20 @@ public Result closeDurableCqs(@CliOption(key = CliStrings.CLOSE_DURABLE_CQS__DUR
...
+      return ResultModel.createError(CliStrings.NO_MEMBERS_FOUND_MESSAGE);
     }
-    return result;
+
+    String[] params = new String[2];
+    params[0] = durableClientId;
+    params[1] = cqName;
+
+    final ResultCollector<?, ?> rc =
+        CliUtil.executeFunction(new CloseDurableCqFunction(), params, targetMembers);
```
Comment (jinmeiliao) [inline geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/CloseDurableCQsCommand.java]: I think we are trying to get away from using CliUtils in the command

File: geode-cq/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/DurableClientCommandsDUnitTest.java [hunk 2]
```diff
@@ -75,17 +82,21 @@
   public void setup() throws Exception {
     int jmxPort = AvailablePortHelper.getRandomAvailableTCPPort();
     Properties props = new Properties();
-    props.setProperty(LOG_LEVEL, "fine");
     props.setProperty(ConfigurationProperties.JMX_MANAGER_HOSTNAME_FOR_CLIENTS, "localhost");
     props.setProperty(ConfigurationProperties.JMX_MANAGER_PORT, "" + jmxPort);
```
Comment (jinmeiliao) [inline geode-cq/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/DurableClientCommandsDUnitTest.java]: I don't think these properties are needed.

File: geode-cq/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/DurableClientCommandsDUnitTest.java [hunk 3]
```diff
@@ -112,9 +124,40 @@ public void testListDurableClientCqs() {
...
+  }
+
+  @Test
+  public void testListDurableClientCqsWhenNoneExist() {
+    CommandStringBuilder csb = new CommandStringBuilder(CliStrings.LIST_DURABLE_CQS);
+    csb.addOption(CliStrings.LIST_DURABLE_CQS__DURABLECLIENTID, CLIENT_NAME);
+    String commandString = csb.toString();
+
+    gfsh.executeAndAssertThat(commandString).statusIsError()
+        .tableHasColumnWithExactValuesInAnyOrder("CQ Name",
```
Comment (jinmeiliao) [inline geode-cq/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/DurableClientCommandsDUnitTest.java]: I think we should deprecate these methods in CommandResultAssert. these are used for the old CommandResult, for the ResultModel, the other assertions are much easier to understand.

File: geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/CloseDurableCQsCommand.java (hunk 1 above)
Comment (jdeppe-pivotal) [reply geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/CloseDurableCQsCommand.java]: Fixed

File: geode-cq/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/DurableClientCommandsDUnitTest.java (hunk 2 above)
Comment (jdeppe-pivotal) [reply geode-cq/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/DurableClientCommandsDUnitTest.java]: Yes - removed.

File: geode-cq/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/DurableClientCommandsDUnitTest.java (hunk 3 above)
Comment (jdeppe-pivotal) [reply geode-cq/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/DurableClientCommandsDUnitTest.java]: Fixed

**Pull request merged**